Index: environment_cfi.py
===================================================================
RCS file: /cvs_server/repositories/CMSSW/CMSSW/DQM/Integration/python/test/environment_cfi.py,v
retrieving revision 1.7
retrieving revision 1.6
diff -u -r1.7 -r1.6
--- environment_cfi.py	9 Feb 2010 11:29:06 -0000	1.7
+++ environment_cfi.py	9 Nov 2009 12:35:55 -0000	1.6
@@ -2,18 +2,20 @@
 
 from DQMServices.Core.DQM_cfg import *
 
-DQM.collectorHost = 'dqm-prod-local.cms'
+DQM.collectorHost = 'dqm-integration.cms'
 DQM.collectorPort = 9090
 
 from DQMServices.Components.DQMEnvironment_cfi import *
 
 dqmSaver.convention = 'Online'
 dqmSaver.referenceHandling = 'all'
-dqmSaver.dirName = '/home/dqmprolocal/output'
-dqmSaver.producer = 'DQM'
-dqmSaver.saveByTime = 1
+dqmSaver.dirName = '/home/dqmdevlocal/output'
+dqmSaver.producer = 'Playback'
+dqmSaver.version = 2
+dqmSaver.runIsComplete = True
+dqmSaver.saveByTime = 1
 dqmSaver.saveByLumiSection = -1
-dqmSaver.saveByMinute = 8
+dqmSaver.saveByMinute = 4
 dqmSaver.saveByRun = -1
 dqmSaver.saveAtJobEnd = False
 
Index: inputsource_cfi.py
===================================================================
--- inputsource_cfi.py	9 Feb 2010 14:52:53 -0000
+++ inputsource_cfi.py	22 Nov 2009 18:37:43 -0000
@@ -20,7 +20,7 @@
     consumerName = cms.untracked.string('DQM Source'),
     SelectHLTOutput = cms.untracked.string('hltOutputDQM'),
     max_queue_depth = cms.int32(5),
-    maxEventRequestRate = cms.untracked.double(100.0),
+    maxEventRequestRate = cms.untracked.double(10.0),
     SelectEvents = cms.untracked.PSet(
         SelectEvents = cms.vstring('*')
     ),
