Index: environment_cfi.py
===================================================================
RCS file: /cvs_server/repositories/CMSSW/CMSSW/DQM/Integration/python/test/environment_cfi.py,v
retrieving revision 1.11
diff -u -p -r1.11 environment_cfi.py
--- environment_cfi.py	11 May 2011 09:52:20 -0000	1.11
+++ environment_cfi.py	12 May 2011 12:15:07 -0000
@@ -2,15 +2,15 @@ import FWCore.ParameterSet.Config as cms
 
 from DQMServices.Core.DQM_cfg import *
 
-DQM.collectorHost = 'dqm-prod-local.cms'
+DQM.collectorHost = 'dqm-integration.cms'
 DQM.collectorPort = 9090
 
 from DQMServices.Components.DQMEnvironment_cfi import *
 
 dqmSaver.convention = 'Online'
 dqmSaver.referenceHandling = 'all'
-dqmSaver.dirName = '/home/dqmprolocal/output'
-dqmSaver.producer = 'DQM'
+dqmSaver.dirName = '/home/dqmdevlocal/output'
+dqmSaver.producer = 'Playback'
 dqmSaver.saveByTime = 1
 dqmSaver.saveByLumiSection = -1
 dqmSaver.saveByMinute = 8
Index: inputsource_cfi.py
===================================================================
RCS file: /cvs_server/repositories/CMSSW/CMSSW/DQM/Integration/python/test/inputsource_cfi.py,v
retrieving revision 1.26
diff -u -p -r1.26 inputsource_cfi.py
--- inputsource_cfi.py	29 Apr 2010 12:03:21 -0000	1.26
+++ inputsource_cfi.py	12 May 2011 12:15:07 -0000
@@ -20,7 +20,7 @@ source = cms.Source("EventStreamHttpRead
     consumerName = cms.untracked.string('DQM Source'),
     SelectHLTOutput = cms.untracked.string('hltOutputDQM'),
     max_queue_depth = cms.int32(5),
-    maxEventRequestRate = cms.untracked.double(100.0),
+    maxEventRequestRate = cms.untracked.double(10.0),
     SelectEvents = cms.untracked.PSet(
         SelectEvents = cms.vstring('*')
     ),
